src/App/Entity/User.php

```diff
@@ -66,70 +65,52 @@ class User implements UserInterface, \Serializable, HasPassword, HasSalt
     private $banned;
 
     /**
-     * @ORM\OneToMany(targetEntity="Proposal", mappedBy="mainAuthor")
-     * @Assert\Valid()
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * Stores the name of the image file associated to the user
      */
-    private $mainProposals;
+    private $path;
 
     /**
-     * @ORM\OneToMany(targetEntity="ProposalDraft", mappedBy="mainAuthor")
-     * @Assert\Valid()
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * Stores temporarily the name of the old image file to delete it after the upload
      */
-    private $mainProposalDrafts;
+    private $temp;
```

Comment:
I don't understand. Why would you store temporarily this in the database?

Reply:
This is used to manage the image file (avatar)